About.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button } from 'react-native';
+import { Font } from 'expo';
+import Logo from './Logo';
+
+
+export class About extends React.Component {
+  state = {fontLoaded: false};
```

Comment:
Make sure to insert spaces around items in curly brackets: `{ fontLoaded: false }`